packages/react-components/react-drawer/stories/src/Drawer/DrawerPosition.stories.tsx

```diff
@@ -71,7 +57,7 @@ export const Position = () => {
               />
             }
           >
-            {setTitle(position)} Drawer
+            {capitalize(position!)} Drawer
```

Comment:
hmm not sure about the `!` usage here , how come that starts can be undefined when we set the default value to `start` ?